python-backend/tests/factories.py

```diff
@@ -430,7 +428,6 @@ class Meta:
     email = factory.Faker('email')
     phone_no = factory.Faker('numerify', text='###-###-####')
     last_logon = TODAY
-    idir_user_detail = factory.SubFactory('tests.factories.IdirUserDetailFactory')
 
     @factory.post_generation
     def idir_user_detail(obj, create, extracted, **kwargs):
```

Comment:
idir_user_detail is one-to-one with core_user, this @factory.post_generation is unnecessary. idir_user_detail should be a RelatedFactory here, and there should be a corresponding core_user = factory.SubFactory on IdirUserDetailsFactory.

Currently, if you tried to call IdirUserDetailFactory(), it would fail because factory.SelfAttribute won't work like that.

Reply:
Oh, I thought you just said that `RelatedFactory` is unreleased, but I see it was `RelatedFactoryList`.

And yes, I originally wrote core_user = factory.SubFactory on IdirUserDetailsFactory, but it was causing problems when being called via the CoreUser factory. I can revisit it, though that's really not the purpose of this PR.